leshan-core-cf/src/main/java/org/eclipse/leshan/core/californium/CoapAsyncRequestObserver.java

```diff
@@ -192,4 +192,4 @@ private synchronized void cancelCleaningTask() {
         }
         cancelled = true;
     }
-}
\ No newline at end of file
+}
```

Comment:
Sounds to related to the fix.

AFAIK, there is no formatting rules about that currently if you think we need to add one let me know.